src/connectedk8s/azext_connectedk8s/vendored_sdks/models/connected_cluster_patch_py3.py

```diff
@@ -17,18 +17,17 @@ class ConnectedClusterPatch(Model):
 
     :param tags: Resource tags.
     :type tags: dict[str, str]
-    :param agent_public_key_certificate: Base64 encoded public certificate
-     used by the agent to do the initial handshake to the backend services in
-     Azure.
-    :type agent_public_key_certificate: str
+    :param properties: Describes the connected cluster resource properties
+     that can be updated during PATCH operation.
+    :type properties: object
     """
 
     _attribute_map = {
         'tags': {'key': 'tags', 'type': '{str}'},
-        'agent_public_key_certificate': {'key': 'properties.agentPublicKeyCertificate', 'type': 'str'},
```

Comment:
why is this removed?

Reply:
This is as per auto-generated from GA swagger. Discussed offline.